acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/condition/eth/EthConditions.java

```diff
@@ -50,4 +52,22 @@ public Condition sendRawTransactionExceptional(
     return new ExpectEthSendRawTransactionException(
         transactions.sendRawTransactionTransaction(transactionData), expectedMessage);
   }
+
+  public Condition expectSuccessfulTransactionReceiptWithReason(
```

Comment:
Why not create a wrapper object for the configuration (`hostName` & `jsonRpcHttpSocketPort`) and use Varargs?

Reply:
done. no longer needed